TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -708,36 +707,39 @@ def makeTreeFromMiniAOD(
     
     # AK8 jet variables - separate instance of jet properties producer
     from TreeMaker.Utils.jetproperties_cfi import jetproperties
+    #JetAK8Tag = cms.InputTag('slimmedJetsAK8')
     process.JetsPropertiesAK8 = jetproperties.clone(
         JetTag       = JetAK8Tag,
         properties = cms.vstring(
             "prunedMass"    ,
             "NsubjettinessTau1"    ,
             "NsubjettinessTau2"    ,
             "NsubjettinessTau3"    ,
-            "bDiscriminatorSubjet1",
-            "bDiscriminatorSubjet2",
+            #"bDiscriminatorSubjet1",
+            #"bDiscriminatorSubjet2",
             "bDiscriminatorCSV"    ,
             "NumBhadrons"          ,
             "NumChadrons"          ,
         )
     )
-    #specify userfloats
-    process.JetsPropertiesAK8.prunedMass = cms.vstring('ak8PFJetsCHSPrunedMass')
-    process.JetsPropertiesAK8.NsubjettinessTau1 = cms.vstring('NjettinessAK8:tau1')
-    process.JetsPropertiesAK8.NsubjettinessTau2 = cms.vstring('NjettinessAK8:tau2')
-    process.JetsPropertiesAK8.NsubjettinessTau3 = cms.vstring('NjettinessAK8:tau3')
-    process.JetsPropertiesAK8.bDiscriminatorSubjet1 = cms.vstring('SoftDrop','pfCombinedInclusiveSecondaryVertexV2BJetTags')
-    process.JetsPropertiesAK8.bDiscriminatorSubjet2 = cms.vstring('SoftDrop','pfCombinedInclusiveSecondaryVertexV2BJetTags')
-    process.JetsPropertiesAK8.bDiscriminatorCSV = cms.vstring('pfBoostedDoubleSecondaryVertexAK8BJetTags')
+    #specify userfloat
+    #process.JetsPropertiesAK8.prunedMass = cms.vstring('prunedMass')
+    process.JetsPropertiesAK8.prunedMass = cms.vstring('ak8PFJetsCHSValueMap:ak8PFJetsCHSPrunedMass')
+    process.JetsPropertiesAK8.NsubjettinessTau1 = cms.vstring('ak8PFJetsCHSValueMap:NjettinessAK8CHSTau1')
```

Comment:
I don't think these are right. It looks like the AK8 jet collection is now Puppi by default with PFCHS info stored as userfloats (reverse of the previous setup). So I think you want to use `NjettinessAK8Puppi:tau1`, etc.

(I turned on the debug flag in `JetProperties` and got this output:
```
subjetCollectionNames: SoftDropPuppi 
btagDiscriminatorNames: pfJetBProbabilityBJetTags pfJetProbabilityBJetTags pfTrackCountingHighEffBJetTags pfSimpleSecondaryVertexHighEffBJetTags pfSimpleInclusiveSecondaryVertexHighEffBJetTags pfCombinedSecondaryVertexV2BJetTags pfCombinedInclusiveSecondaryVertexV2BJetTags softPFMuonBJetTags softPFElectronBJetTags pfCombinedMVAV2BJetTags pfCombinedCvsLJetTags pfCombinedCvsBJetTags pfDeepCSVJetTags:probb pfDeepCSVJetTags:probc pfDeepCSVJetTags:probudsg pfDeepCSVJetTags:probbb pfBoostedDoubleSecondaryVertexAK8BJetTags pfBoostedDoubleSecondaryVertexAK8BJetTags 
userFloatNames: NjettinessAK8Puppi:tau1 NjettinessAK8Puppi:tau2 NjettinessAK8Puppi:tau3 ak8PFJetsCHSValueMap:NjettinessAK8CHSTau1 ak8PFJetsCHSValueMap:NjettinessAK8CHSTau2 ak8PFJetsCHSValueMap:NjettinessAK8CHSTau3 ak8PFJetsCHSValueMap:ak8PFJetsCHSPrunedMass ak8PFJetsCHSValueMap:ak8PFJetsCHSSoftDropMass ak8PFJetsCHSValueMap:eta ak8PFJetsCHSValueMap:mass ak8PFJetsCHSValueMap:phi ak8PFJetsCHSValueMap:pt ak8PFJetsPuppiSoftDropMass 
userIntNames: 
```
)